control-logger.cabal

```diff
@@ -17,7 +17,6 @@ library
   exposed-modules:     Control.Logger.Internal
   build-depends:       base
                      , aeson
-                     , bloodhound
```

Comment:
You have to drop kati-elasticsearch also. It depends on bloodhound by self.

Reply:
Oh. indeed. dropped now.